Kernel/Process.h

```diff
@@ -713,6 +713,37 @@ class Process
         FileDescriptionAndFlags* m_description { nullptr };
     };
 
+    class ProcessProcFSTraits : public ProcFSExposedComponent {
+    public:
+        static KResultOr<NonnullRefPtr<ProcessProcFSTraits>> create(Badge<Process>, WeakPtr<Process> process)
```

Comment:
Please call this `try_create()` since it may fail.

Reply:
All functions which were named `create*` and return a `KResultOr` were updated to be `try_create*`.

Reply:
Whoops, missed the actual one you pointed out. :^)